index.html

```diff
@@ -18,25 +18,26 @@
     <div class="mainPage">
 
         <!-- логотип и телефон -->
-        <div class="mainPageheader">
+        <div class="mainPageheader mt25 mb3">
             <img class="logo" src="./images/HELYFLY.jpg" alt="logotype">
 
             <div class="phone">Телефон:</div>
-            <div class="numberPhone">8(800)-123-12-12</div>
+            <p><a class="numberPhone" href="tel:8(800)-123-12-12">8(800)-123-12-12</a></p>
```

Comment:
In my opinion, we don't need <p /> element to wrap <a /> here, we can simply use <a /> with display: block